src/Voron/Data/VoronStream.cs

```diff
@@ -114,10 +131,13 @@ public override int ReadByte()
                     return -1;
             }
 
-            var page = _llt.GetPage(chunk.PageNumber, _pagerRef);
-            _pagerRef.Pager.EnsureMapped(_llt, _pagerRef.PagerPageNumber, VirtualPagerLegacyExtensions.GetNumberOfOverflowPages(page.OverflowSize));
```

Comment:
Since `AcquirePagePointerWithOverflowHandling` takes care of it we are fine. However it won't be called if we cake it from page locator's cache:
https://github.com/ravendb/ravendb/blob/v4.0/src/Voron/Impl/LowLevelTransaction.cs#L442-L443

Not sure if that is a problem then.